### PR TITLE
[DRAFT] "Start Building" Section

### PR DESCRIPTION
**Pull request by ryanRfox**
This PR includes content updates for
- build-apps/setup/
- build-apps/connect/
- build-apps/hello-world/

Changes include
- Replacing /v1 endpoints with /v2
  - Update calls and responses as needed
  - Update test to reflect changes
- Updating SDK and CLI code snippets with /v2 clients

### CODE REVIEW

**Comment by ryanRfox** [comment]
This work in progress and includes many "TODO:" tags to remind to research further and will  be updated in future commits.

File: docs/build-apps/connect.md [hunk 1]
```diff
@@ -291,27 +302,21 @@ Next consensus protocol: [LINK_TO_FUTURE_PROTOCOL_SPEC]
...
 Next consensus protocol supported: [true|false]
 Has Synced Since Startup: [true|false]
+Last Catchpoint: []
 Genesis ID: [GENESIS_ID]
-Genesis hash: [BASE64_GENESIS_HASH]
+Genesis hash: [GENESIS_HASH]
 ```
 
-Check that the `genesis_id` and the `genesis_hash_b64`, as shown in the REST response below, match your chosen network before proceeding.
+Check that the `genesis-id` and the `genesis-hash`, as shown in the REST response below. Ensure both match your chosen network before proceeding.
```
Comment (ejbaran) [inline docs/build-apps/connect.md:310]: Remove "that" form "check that"

File: docs/build-apps/connect.md [hunk 2]
```diff
@@ -291,27 +302,21 @@ Next consensus protocol: [LINK_TO_FUTURE_PROTOCOL_SPEC]
...
-        "major": 2,
-        "minor": 0,
-        "build_number": 2,
-        "commit_hash": "1bc1d4c9",
-        "branch": "rel/stable",
-        "channel": "stable"
-    }
+    "consensus-version": "https://github.com/algorandfoundation/specs/tree/4a9db6a25595c6fd097cf9cc137cc83027787eaa",
+    "fee": 1,
+    "genesis-hash": "TODO:betanet?",
```
Comment (ejbaran) [inline docs/build-apps/connect.md:316]: Why not testnet?

File: docs/build-apps/connect.md [hunk 3]
```diff
@@ -10,171 +10,137 @@ Instantiate an **algod** client with your preferred SDK.
 ```JavaScript tab=
 const algosdk = require('algosdk');
 
-async function connectToNetwork() {
+const algodToken = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
+const algodServer = "http://localhost";
+const algodPort = 8888;
```
Comment (ejbaran) [inline docs/build-apps/connect.md:15]: Why don't we use the default port with sandbox. Isn't it 4001?

File: docs/build-apps/connect.md [hunk 4]
```diff
@@ -203,46 +168,54 @@ Next consensus protocol: [LINK_TO_FUTURE_PROTOCOL_SPEC]
...
 Next consensus protocol supported: [true|false]
 Has Synced Since Startup: [true|false]
+Last Catchpoint: []
 Genesis ID: [GENESIS_ID]
-Genesis hash: [BASE64_GENESIS_HASH]
+Genesis hash: [GENESIS_HASH]
 ```
 
-The _status_ methods return information about the status of the node, such as the latest round<LINK TO GLOSSARY>, referred to as `lastRound`, from the perspective of the node you are connected to. Each of the SDKs may differ slightly in which information they return for each call. Shown below is the response from the REST API call.
+The _status_ methods returns information about the status of the node, such as the latest round<LINK TO GLOSSARY>, referred to as `lastRound`, from the perspective of the node you are connected to. Each of the SDKs may differ slightly in which information they return for each call. Shown below is the response from the REST API call.
```
Comment (ejbaran) [inline docs/build-apps/connect.md:176]: remove <LINK TO GLOSSARY> and update `lastRound` to `last-round` as per the new response structure.

File: docs/build-apps/connect.md [hunk 5]
```diff
@@ -251,34 +224,34 @@ The _version_ methods return information about the identity of the network and t
...
-	fmt.Printf("%s\n", versionJSON)
+	fmt.Printf("%s\n", txParams)
 ...
 ```
 
 ```bash tab="cURL"
 curl -i -X GET \
    -H "X-Algo-API-Token:<algod-token>" \
- 'http://<algod-address>:<algod-port>/versions'
+ 'http://<algod-address>:<algod-port>/v2/transactions/params'
```
Comment (ejbaran) [inline docs/build-apps/connect.md:254]: Update with sandbox address and port?

File: docs/build-apps/connect.md [hunk 6]
```diff
@@ -291,268 +264,20 @@ Next consensus protocol: [LINK_TO_FUTURE_PROTOCOL_SPEC]
...
 Genesis ID: [GENESIS_ID]
-Genesis hash: [BASE64_GENESIS_HASH]
+Genesis hash: [GENESIS_HASH]
 ```
 
-Check that the `genesis_id` and the `genesis_hash_b64`, as shown in the REST response below, match your chosen network before proceeding.
+Check the `genesis-id` and the `genesis-hash`, as shown in the REST response below. Ensure both match your chosen network before proceeding.
 
-```json hl_lines="5 6" tab="Response"
+```json hl_lines="4 5" tab="Response"
```
Comment (ejbaran) [inline docs/build-apps/connect.md:274]: Can we change TestNet.

File: docs/build-apps/hello_world.md [hunk 7]
```diff
@@ -82,71 +76,121 @@ Private key mnemonic: [PASSPHRASE]
...
+## Connect your client
+
+Each SDK provides a client which must instantiate prior to making calls to the API endopoints. You must provide values for `<algod-taddress>`, `<port>` and `<algod-token>`. The CLI Tools implement the client natively. 
+
+_Learn more about [Connecting to a Node](connect.md)._
+
+```JavaScript tab=
+const algodToken = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
+const algodServer = "http://localhost";
+const algodPort = 8888;
```
Comment (ejbaran) [inline docs/build-apps/hello_world.md:96]: Same comment about port being 4001.

File: docs/build-apps/hello_world.md [hunk 8]
```diff
@@ -82,71 +76,121 @@ Private key mnemonic: [PASSPHRASE]
...
+
 ## Add funds
 For [TestNet](../../reference/algorand-networks/testnet/#faucet) and [BetaNet](../../reference/algorand-networks/betanet/#faucet), copy and paste the public portion of your key pair in the corresponding faucet prompt and click "Submit". A `200` response means the transaction went through and your balance increased by 100,000,000 microAlgos (i.e. 100 Algos).
 
 !!! info
 	Amounts are returned in microAlgos - the base unit for Algos. Micro denotes a unit x 10^-6. Therefore, 1 Algo equals 1,000,000 microAlgos.
 
+## Connect your client
+
+Each SDK provides a client which must instantiate prior to making calls to the API endopoints. You must provide values for `<algod-taddress>`, `<port>` and `<algod-token>`. The CLI Tools implement the client natively.
```
Comment (ejbaran) [inline docs/build-apps/hello_world.md:89]: typo: algod-taddress

Comment (ejbaran) [inline docs/build-apps/hello_world.md:89]: Tools --> tools

**Comment by ryanRfox** [comment]
Replaced by #14

**Pull request closed without merging**